docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-immutable-monitoring-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='environment-immutable-monitoring-proc']
+= Deploying monitoring and Smart Router for an environment with immutable servers
+
+If you want to use {CENTRAL} Monitoring and Smart Router for an environment with immutable servers, you must deploy them before deploying any {KIE_SERVERS}. If you do not want to use these components, skip this procedure.
+
+To deploy {CENTRAL} Monitoring and Smart Router for an environment with immutable servers, use the `rhpam70-prod-immutable-monitor.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam70-prod-immutable-monitor.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/rhpam70-prod-immutable-monitor.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-path>` with the path to the downloaded template file.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and Smart Router.
```

Comment:
It is (also and more importantly) used to create a distinct set of deployment configs, services, routes, labels and artifacts and distinguish different applications built using the same or similar templates within the same project.